runtime/oti/j9nonbuilder.h

```diff
@@ -5638,6 +5655,8 @@ typedef struct J9JavaVM {
 	struct J9HashTable* ensureHashedClasses;
 #if JAVA_SPEC_VERSION >= 19
 	U_64 nextTID;
+	J9VMContinuation *continuationList;
+	omrthread_monitor_t continuationListMutex;
 #endif /* JAVA_SPEC_VERSION >= 19 */
```

Comment:
GC will maintain arrayLists per region for Continuation, we don't need the global continuation list and the mutex, GC will provide an iterator for retrieving the list of J9VMContinuation.

Reply:
the vm needs this

Reply:
but GC will not maintain the continuationList(can not synchronize it with "Continuation Object" list).

Reply:
correct, this list is not for the GC